misc/packaging/README.md

```diff
@@ -0,0 +1,117 @@
+# ubuntuSoS
```

Comment:
For inclusion in our documentation, this needs to be written as restructureText as is all our other documentation.  Look at one of the other README.rst files in the misc folders.  The doc will also need to be added to the table of contents once it's ready.

Reply:
change to rst